### DISCUSSION TITLE
360° preview not working

### DISCUSSION THREAD

**Discussion by AndresSalomon1990** [google/model-viewer#2981] [Q&A]
Hello, I'm testing a model and I found that in my phone (Nokia 7.1 - Android 10) the 360° preview is not working. What is strange is that the AR works perfectly. Also it works well on another Android devices.

I attach a snapshot and you can see a sad face on the top left of the model viewer frame. What can be causing this? I don't have this problem with other projects I have worked with Model Viewer. Could be the model (it's about 10mb)? Something with my phone?

I appreciate any insights! Thanks!

![preview not working](https://user-images.githubusercontent.com/69881468/141841960-4716cbce-1737-4d3f-9f98-d6651ff5dde3.png)

**Comment by ghost** [comment]
This screen shot is looks like its from SceneViewer is that right? Can you share the model? Can you test the model in the Model-viewer editor?

**Reply by AndresSalomon1990** [reply]
Yes sir, I'm using SceneViewer as default for this project. I can't share the model (could have problems with the client), but I tried it in the Model-Viewer editor and I only saw a warning (snap below - don't know what is). Also my 3D editors teammates are trying some improvements on the model to see what could be going on. I'm know thinking that the problem could be the model and not SceneViewer, but would be good to know if something like this has happened.

I will make an update after I can talk with the team tomorrow!

![warning](https://user-images.githubusercontent.com/69881468/142093254-c72bf66b-2a4d-4368-a388-c2ea7940e072.png)
.
